tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -623,6 +624,45 @@ test.serial('validate file properties', async (t) => {
   );
 });
 
+test.serial('when workflow_start_time is provided, then createdAt is set to workflow_start_time', async (t) => {
+  t.context.event.config.provider = {
+    id: 'MODAPS',
+    protocol: 'http',
+    host: '127.0.0.1',
+    port: 3030,
+  };
+  const workflowStartTime = 1636334502146;
+  t.context.event.config.workflowStartTime = workflowStartTime;
+
+  const output = await syncGranule(t.context.event);
+
+  t.is(output.granules.length, 1);
+  output.granules.forEach((g) => {
+    t.true(Number.isInteger(g.createdAt));
+    t.is(g.createdAt, workflowStartTime);
+  });
+});
+
+test.serial('when workflow_start_time is NOT provided, then createdAt is set to Date.now()', async (t) => {
+  t.context.event.config.provider = {
+    id: 'MODAPS',
+    protocol: 'http',
+    host: '127.0.0.1',
+    port: 3030,
+  };
+  const now = Date.now();
+
+  sinon.stub(Date, 'now').returns(now);
```

Comment:
```suggestion
  const nowStub = sinon.stub(Date, 'now').returns(now);
  t.teardown(() => nowStub.restore());
```

You need to make sure to revert the stub when you're done